README.md

```diff
@@ -1,8 +1,27 @@
-Confluent's Apache Kafka client for Python
-==========================================
+Confluent's Python Client for Apache Kafka<sup>TM</sup>
```

Comment:
in fact this should be capitalized (it's a heading) and words in other headings on this page should also be capitalized.